fmn/rules/anitya.py

```diff
@@ -35,6 +35,36 @@ def anitya_catchall(config, message):
     return message['topic'].split('.')[3] == 'anitya'
 
 
+@hint(categories=['anitya'], invertible=False)
+def anitya_specific_distro(config, message, distro=None, *args, **kw):
+    """ Distro-specific release-monitoring.org events
+
+    This rule will match all anitya events *only for a particular distro*.
+    """
+
+    if not distro:
+        return False
+
+    if not anitya_catchall(config, message):
+        return False
+
+    d = message['msg'].get('distro', {})
+    if d:  # Have to be careful for None here
+        if d.get('name') == distro:
```

Comment:
I do it in anitya itself (admin can edit the distro names to make sure the cast is correct, users can enter any cast they want, it'll be fixed), that's why I was mentioning it, since it's user input I thought it might be nice to be generous and support `Fedora` as well as `fedora` :)